ple/games/donkeykong/DonkeyKong.py

```diff
@@ -0,0 +1,103 @@
+__author__ = 'Erilyth'
+import pygame
+from Person import Person
+import random
+
+'''
+This class defines all the Donkey Kongs present in our game.
+Each donkey kong can only move on the top floor and cannot move vertically.
+'''
+
+
+class DonkeyKong(Person):
```

Comment:
wont this file and the class DonkeyKong interfere with loading the game as follows: `from ple.games.donkeykong import DonkeyKong`